Add deprecated software to "deprecated software" page (#5971)
ethereum/ethereum-org-website: src/content/deprecated-software/index.md

[Remix](https://remix.ethereum.org/) is a browser-hosted IDE for Solidity / smart contract development, testing, and deployment. It also has a desktop option.

> ## Minimal

Reviewer: ```suggestion
### Minimal {#minimal}
```

Deprecated in 2020.

**Summary**

Minimal was a modular implementation of the Ethereum blockchain written in Go.

**Archives**

> [Archived Github repo](https://github.com/umbracle/minimal)

Reviewer: ```suggestion
[Archived Github repo](https://github.com/umbracle/minimal)
```

**History**

Minimal was replaced by [polgon-sdk](https://github.com/0xPolygon/polygon-edge)

> ### Mana-Ethereum

Reviewer: ```suggestion
### Mana-Ethereum {#mana-ethereum}
```

**Summary**

> Mana was an Ethereum client built using Elixir.

Reviewer: ```suggestion
Mana-Ethereum was an Ethereum client built using Elixir.
```

**Archives**

> [Archived Github repo](https://github.com/mana-ethereum/mana)

Reviewer: ```suggestion
[Archived GitHub repo](https://github.com/mana-ethereum/mana)
```

**History**

> Mana's Github repository has not been explicitly archived but the last commit was in 2019.

Reviewer: Created an issue https://github.com/mana-ethereum/mana/issues/767

> Mana's Github repository has not been explicitly archived but the last commit was in 2019.

Reviewer: ```suggestion
Mana-Ethereum's Github repository has not been explicitly archived, but the last commit was in 2019.
```

> Mana-Ethereum's Github repository has not been explicitly archived, but the last commit was in 2019.

Reviewer: ```suggestion
Mana-Ethereum's GitHub repository has not been explicitly archived, but the last commit was in 2019.
```

### Aleth (cpp-ethereum) {#aleth}

Deprecated on October 6, 2021

**Summary**

> Aleth was an Ethereum client written in C++,originally known as cpp-ethereum.

Reviewer: ```suggestion
Aleth (formerly known as cpp-ethereum) was an Ethereum client written in C++.
```

[Geth](https://geth.ethereum.org/) is a well-known alternative Ethereum client.

> ### Ethereum-H

Reviewer: ```suggestion
### Ethereum-H {#ethereum-h}
```

**Archives**

> The Ethereum-H archives have been removed from Github.

Reviewer: ```suggestion
The Ethereum-H archives have been removed from GitHub.
```

**History**

Ethereum-H was an Ethereum client written in Haskell. It was deprecated around 2015.

**Alternatives**

[Geth](https://geth.ethereum.org/), [Nethermind](http://nethermind.io/), [Besu](https://pegasys.tech/solutions/hyperledger-besu/) and [Erigon](https://github.com/ledgerwatch/erigon) are viable alternative Ethereum clients - there is no current Haskell client.

> ### ruby-ethereum

Reviewer: ```suggestion
### ruby-ethereum {#ruby-ethereum}
```

**Archives**

> [ruby-ethereum Github page](https://github.com/cryptape/ruby-ethereum)

Reviewer: ```suggestion
[ruby-ethereum GitHub repo](https://github.com/cryptape/ruby-ethereum)
```

**History**

ruby-ethereum was an Ethereum client written in Ruby. It was deprecated around 2018.

**Alternatives**

> [Geth](https://geth.ethereum.org/), [Nethermind](http://nethermind.io/), [Besu](https://pegasys.tech/solutions/hyperledger-besu/) and [Erigon](https://github.com/ledgerwatch/erigon) are viable alternative Ethereum clients - there is no current Ruby client.

Reviewer: ```suggestion
[Geth](https://geth.ethereum.org/), [Nethermind](http://nethermind.io/), [Besu](https://pegasys.tech/solutions/hyperledger-besu/) and [Erigon](https://github.com/ledgerwatch/erigon) are viable alternative Ethereum clients. There is no current Ruby client.
```
